README.md

```diff
@@ -42,3 +42,10 @@ For example, if the instance connection name is `foo:bar:baz`, the JDBC URL
 would be 
 `jdbc:mysql://google/mydb?cloudSqlInstance=foo:bar:baz&socketFactory=com.google.cloud.sql.mysql.SocketFactory`
 
+## Credentials
```

Comment:
This comment sent me down a rabbit hole of trying to figure out what the right way to do default credentials on k8s is.

Do you have a link to some documentation on this?